lib/grape-swagger/endpoint.rb

```diff
@@ -227,11 +227,11 @@ def apply_success_codes(route)
       [default_code]
     end
 
-    def tag_object(route)
+    def tag_object(route, path)
       version = GrapeSwagger::DocMethods::Version.get(route)
       version = [version] unless version.is_a?(Array)
       Array(
-        route.path.split('{')[0].split('/').reject(&:empty?).delete_if do |i|
+        path.split('{')[0].split('/').reject(&:empty?).delete_if do |i|
```

Comment:
This array initialisation is quite complex and worth abstracting away from Application logic.